pom.xml

```diff
@@ -1339,5 +1339,27 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>disableWizard</id>
+      <activation>
+        <property>
+          <name>!enableWizard</name>
+        </property>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.jenkins-ci.tools</groupId>
+            <artifactId>maven-hpi-plugin</artifactId>
+            <configuration>
+              <systemProperties>
+                <name>hudson.Main.development</name>
+                <value>true</value>
```

Comment:
And the bonus of my suggestion is that it mirrors exactly the system property people should set when launching the .war file standalone... if you want to add a profile to set the property to `false` you can... but that is death by a thousand profiles and an anti-pattern IMHO